mission/src/views/ItemInfo.vue

```diff
@@ -0,0 +1,250 @@
+<template>
+<div id='item-info-page'>
+  <div class="itemImg">
+      <img class="itemImg" src="@/assets/itemImg.png">
+  </div>
+  <div class="sellerInfo">
+    <img class="sellerImg" src="@/assets/sellerImg.png">
+    <img class="bookmarkImg" src="@/assets/bookmarkImg.png">
+    <div class="sellerName">
+    코트맛집
+    </div>
+    <br>
+    <div class="hashtag">
+    #남성&nbsp;&nbsp;&nbsp;#코트
+    </div>
+  </div>
+  <div class="itemContent">
+    <h2 class="title">세미 오버핏 트렌치 코트</h2>
+    <div class="priceInfo">
+      <span class="discountRate">{{ discountRate }}%</span>
+      <span class="price">{{ price }}원</span>
+      <span class="beforePrice">{{beforePrice}}원</span>
+    </div>
+    <div class="itemInfo">
+      <h3 class="itemInfoTitle">상품정보</h3>
+      <div class="explainText1">남친룩의 정석 코디를 위한 트렌치 코트!</div>
+      <img class="itemInfoImg" src="@/assets/itemInfoImg.png">
+      <img class="itemInfoDetailImg" src="@/assets/itemInfoDetailImg.png">
+      <div class="explainText2">
+        - 특유의 여유있는 세미 오버 사이즈 실루엣
+        <br>
+        - 정교한 패턴 작업으로 세심하게 라인을 완성
+        <br>
+        - 간절기 아우터에 적합한 두께감과 뛰어난 착용감
+      </div>
+    </div>
+  </div>
+  <div class="review">
+    <h3 class="reviewTitle">
+      리뷰
+    </h3>
+    <div class="reviewContent1">
+      <img class="reviewImg1" src="@/assets/reviewImg1.png">
+      <span class="reviewerName1">{{ reviewerName1 }}</span>
+      &nbsp;
+      <span class="reviewDat1">{{ reviewDate1 }}</span>
+      <br>
+      <span class="reviewTitle1">{{ reviewTitle1 }}</span>
+      <br>
+      <span class="reviewMessage1">{{ reviewMessage1 }}</span>
+    </div>
+    <div class="reviewContent2">
+      <span class="reviewerName2">{{ reviewerName2 }}</span>
+      &nbsp;
+      <span class="reviewDat2">{{ reviewDate2 }}</span>
+      <br>
+      <span class="reviewTitle2">{{ reviewTitle2 }}</span>
+      <br>
+      <span class="reviewMessage2">{{ reviewMessage2 }}</span>
+    </div>
+  </div>
+  <br><br>
+  <div class="footer">
+    <button class="buyBotton">{{ price }}원</button>
+  </div>
+</div>
+</template>
+
+<script>
+export default {
+  name: 'ItemInfoPage',
+  data() {
+    return {
+      discountRate: 30,
+      price: 140000,
+      beforePrice: 200000,
```

Comment:
```suggestion
price: {
   discountRate: 30,
   price: 140000,
   beforePrice: 200000,
},
```

이런 식으로 관련있는 데이터들을 객체로 묶어 구조화 할 수 있습니다.  
데이터 객체 수가 많을 시,  더 직관적인 설계가 가능합니다. :)

Reply:
데이터를 객체로 묶어서 구조화 할 수 있다는 건 처음 알았습니다! 훨씬 가독성이 좋아지고 나중에 코드 작성하기도 편리할 것 같습니다. 좋은 피드백 감사합니다!

Reply:
아래의 말씀에는 답글이 안눌려서 여기에 말씀 드립니다! 저는 vs code로만 작성하고 저장하면서 실시간으로 확인해봤는데 개발자도구는 원하는 부분을 클릭해서 해당 코드가 위치한 곳만 참고하는 용도로만 사용했었습니다,,
개발자도구로 미리 style을 미리 적용할 수 있다니,,, 처음 알았네요,, 참고해서 다음 미션 수행할 때 적용해서 훨씬 편리하게 개발할 수 있을 것 같습니다! 감사합니다! 😀